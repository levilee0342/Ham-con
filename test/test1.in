-8648
-27800
-18530
-22050
-2072
-25831
-10615
-29092
-8978
-20020
-13983
-1983
-24010
-1457
-15606
