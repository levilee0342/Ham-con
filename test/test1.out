-1457
-29092